xbmc/interfaces/legacy/ModuleXbmc.h

```diff
@@ -462,6 +462,33 @@ namespace XBMCAddon
     String getInfoImage(const char * infotag);
 #endif
 
+#ifdef DOXYGEN_SHOULD_USE_THIS
+    ///
+    /// \ingroup python_xbmc
+    /// @brief \python_func{ xbmc.getdatabasename(dbtype) }
+    /// Get full database name of the supplied type currently in use.
+    ///
+    /// @param name                  string - database type of the full
+    ///                              database name you want returned.
+    /// @return                      The full database name currently in use.
+    /// @note Database types can be: addon, epg, music, pvr, texture, video
```

Comment:
I think you want a comma after video?
```suggestion
    /// @note Database types can be: addon, epg, music, pvr, texture, video,
```